MinecraftClient/Resources/config/MinecraftClient.ini

```diff
@@ -247,3 +247,16 @@ items=                            # separate each item name with a comma ',': It
 # /!\ You SHOULD use /replay stop or exit the program gracefully with /quit OR THE REPLAY FILE MAY GET CORRUPT!
 enabled=false
 backupinterval=300                # How long should replay file be auto-saved, in seconds. Use -1 to disable
+
+[WebSocket]
+# Enables you to remotely control the MCC over Web Socket
+# Get real-time data from the client that you could use for external processing and decision making
+# Example usage could be writting chat bots in other languages (although for the time being this is limited in functionality)
+# Technical details:
+#	1. The route for the Web Socket server is: /mcc (Example connection string: "ws://mydomain.com:8043/mcc")
+#	2. If you set a password, you're required to log in with the following command: Authenticate yourpasswordhere
+# (!) it is recommended to protect your Web Scoket instance with a password.
+# /!\ For the time being this should be considered as an experimental feature susceptible to change
+enabled=false
+port=8043                        # Web Socket port (0-65535)
```

Comment:
I'd suggest adding a bind address setting as well and defaulting to `127.0.0.1` to make sure no remote access is possible by default.

Reply:
Yeah, that is a good Idea.

Reply:
Done